core/src/datayoga_core/__init__.py

```diff
@@ -1,6 +1,24 @@
 import logging
 from typing import Any, Dict, List, Optional
 
+try:
+    # This hack makes it possible to use the new driver with SQLAlchemy 1.4.*
+    # More: https://lnk.pw/swif
+
+    import sys
+
+    import oracledb
+    from oracledb.exceptions import DatabaseError
+
+    oracledb.version = "8.3.0"
+    sys.modules["cx_Oracle"] = oracledb
+    try:
+        oracledb.init_oracle_client()
+    except DatabaseError:
+        pass
+except ImportError:
+    pass
```

Comment:
can't we have this snippet as a util so we won't have code duplication and use it here and in the test?

Reply:
No. This code should be executed before other imports. It's necessary.

Reply:
I understand, but why can't be a util? but this is the same issue raise by @zalmane, so please refer it there.